editors/neovim/Portfile

```diff
@@ -49,13 +49,14 @@ configure.args-append   -DUSE_BUNDLED=OFF \
                         -DLUA_PRG=${prefix}/bin/luajit
 
 subport neovim-devel {
-    github.setup    neovim neovim 6fa0f303d7f0823bfc5ba6cc7b4e7a7cd76143ac
-    version         20240101-[string range ${github.version} 0 6]
+    github.setup    neovim neovim 226466a46941cd6b693f56b6eb7f4293d4066c8d
+    version         20240126-[string range ${github.version} 0 6]
     revision        0
 
-    checksums       rmd160  419fefb665764a319a93b27d4c8b838a90c79081 \
-                    sha256  fe74369fc30a32ec7a086b1013acd0eacd674e7570eb1acc520a66180c9e9719 \
-                    size    11590180
+    github.tarball_from tarball
```

Comment:
(Maybe we want `archive` instead of `tarball` in the future.)

Reply:
`neovim` has the following lines (see below). I recently noticed that `neovim-devel` now is a subport. When I tried to update it, instead of the commit hash, the archive was pulled out from the v0.9.5 tag. I don't know why that, maybe this bug in the github PG.
```
github.setup            neovim neovim 0.9.5 v
github.tarball_from     archive
```